cli/src/ios/open.ts

```diff
@@ -1,22 +1,9 @@
 import open from 'open';
 
-import c from '../colors';
 import { wait } from '../common';
 import type { Config } from '../definitions';
-import { fatal } from '../errors';
-
-import { findXcodePath } from './common';
 
 export async function openIOS(config: Config): Promise<void> {
-  const xcodeProject = await findXcodePath(config);
-
-  if (!xcodeProject) {
-    fatal(
-      'Xcode workspace does not exist.\n' +
-        `Run ${c.input('npx cap add ios')} to bootstrap a new iOS project.`,
-    );
-  }
-
-  await open(xcodeProject, { wait: false });
+  await open(config.ios.nativeXcodeWorkspaceDirAbs, { wait: false });
```

Comment:
I think we should check if it exists before trying to open, as open doesn't provide any feedback if Xcode was opened or not

Reply:
I agree, but this should happen automatically when we try to load the value from the config.